service/src/main/java/bio/terra/pipelines/db/entities/PipelineRun.java

```diff
@@ -59,38 +75,60 @@ public PipelineRun(
       UUID jobId,
       String userId,
       Long pipelineId,
+      UUID controlWorkspaceId,
       LocalDateTime created,
       LocalDateTime updated,
       String status,
       String description,
       String resultUrl,
       Boolean isSuccess,
-      String output) {
+      Map<String, String> output) {
     this.jobId = jobId;
     this.userId = userId;
     this.pipelineId = pipelineId;
+    this.controlWorkspaceId = controlWorkspaceId;
     this.created = created;
     this.updated = updated;
     this.status = status;
     this.description = description;
     this.resultUrl = resultUrl;
     this.isSuccess = isSuccess;
-    this.output = output;
+    setOutput(output);
```

Comment:
not sure this is the best way to do this!

Reply:
hmm is there any way we could move this to the PipelineRunService class. Entities should generally be pretty clean